features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapReceiverImpl.java

```diff
@@ -97,6 +107,19 @@ public void setTrapdConfig(TrapdConfiguration newTrapdConfig) {
         }
     }
 
+	@Override
+	public boolean equals(Object obj) {
```

Comment:
@j-white :Sorry there has been mistake during code merge.Some files have got deleted and hence certain changes have not got reflected,Just now i got to know those.I have pushed the new code,Which you can review now.